index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Tehnical Documentation</title>
+</head>
+<body>
+    <nav id="navbar">
+        <header>JS Documentation</header>
+<ul>
+   
+    <li><a class="nav-link" href="#Introduction" >Introduction</a></li>
+    <li><a class="nav-link" href="What_you_should_already_know" >What you should already know</a></li>
+    <li><a class="nav-link" href="JavaScript_and_Java" >JavaScript and Java</a></li>
```

Comment:
Please the browser say that your file couldn't be accessed! You forgot to add the symbol link before the href. For example href= #link to make sure your link is functioning

Reply:
@Piercel2022 Please give the right symbol to your link (#) so that the browser can follow the link as it should be